recipes/winfiletime/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "winfiletime" %}
+{% set version = "2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/winfiletime-{{ version }}.tar.gz
+  sha256: c7c0337ac9ca1af982f623433f93dcbdf5d950eebdf92cd822093311650978ba
+  patches:
+    - 0001-utc.patch
```

Comment:
Can you add a comment on why this patch is needed?

Reply:
Yes, I added it. Version 2.0 includes a wrong import in `__init__.py` that prevents the whole module to be imported.
The patch just remove the superfluous import.
A PR is waiting for merge since more than a year, see https://github.com/jleclanche/winfiletime/pull/1